diff --git a/dist/src-DecVfDJY.mjs b/dist/src-DecVfDJY.mjs
index 1823dfdd2365eca1867d30bc20801b64980f827a..ccda6c6d34a8ffa86360a684f1adaa1116346479 100644
--- a/dist/src-DecVfDJY.mjs
+++ b/dist/src-DecVfDJY.mjs
@@ -561,7 +561,7 @@ async function loadConfigFile(options, workspace) {
 			} else if (stats.isDirectory()) cwd = resolved;
 		}
 	}
-	const nativeTS = process.features.typescript || process.versions.bun || process.versions.deno;
+	const nativeTS = process.versions.bun || process.versions.deno;
 	let { config, sources } = await loadConfig.async({
 		sources: overrideConfig ? [{
 			files: filePath,
